src/components/Tables/Table.tsx

```diff
@@ -3,9 +3,15 @@
 import React, { useState, useEffect } from "react";
 import DataTable, { TableColumn } from "react-data-table-component";
 import TableFilterBar, { FilterText } from "@/components/Tables/TableFilterBar";
+import { styled } from "styled-components";
```

Comment:
should be `import styled from "styled-components";`

surprised it exports styled as a prop as well as the default tbf